### ISSUE TITLE
"app launches" test passes in test-golem-recommended.R but not when run in devtools::check()

### ISSUE THREAD

**Issue by jacksonschan** [ThinkR-open/golem#422]
Hi golem team,

First of all, thanks for developing this wonderful workflow! I just started a new golem project (name = "lzdm") and the "app launches" test when run in the **test-golem-recommended.R** file, but when called using **devtools::check** in **03_deploy.R**, I get the following error: 

`─ Test failures ──────────────────────────────────────── testthat ────

> library(testthat)
> library(lzdm)
> 
> test_check("lzdm")
── 1. Failure: app launches (@test-golem-recommended.R#29)  ───────────
x$is_alive() isn't true.

══ testthat results  ══════════════════════════════════════════════════
[ OK: 2 | SKIPPED: 0 | WARNINGS: 0 | FAILED: 1 ]
1. Failure: app launches (@test-golem-recommended.R#29) 

Error: testthat unit tests failed
Execution halted

1 error x | 0 warnings ✓ | 1 note x
> `

**Session Info**:
`R version 3.6.1 (2019-07-05)
Platform: x86_64-apple-darwin15.6.0 (64-bit)
Running under: macOS Mojave 10.14.6

Matrix products: default
BLAS:   /System/Library/Frameworks/Accelerate.framework/Versions/A/Frameworks/vecLib.framework/Versions/A/libBLAS.dylib
LAPACK: /Library/Frameworks/R.framework/Versions/3.6/Resources/lib/libRlapack.dylib

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods  
[7] base     

other attached packages:
[1] lzdm_0.0.0.9000 golem_0.2.1     shiny_1.4.0.2   testthat_2.3.2 

loaded via a namespace (and not attached):
  [1] colorspace_1.4-1   ellipsis_0.3.0     class_7.3-15      
  [4] rsconnect_0.8.16   leaflet_2.0.3      rgdal_1.4-8       
  [7] rprojroot_1.3-2    fs_1.3.1           rstudioapi_0.11   
 [10] roxygen2_7.1.0     remotes_2.1.1      DT_0.13           
 [13] gh_1.1.0           fansi_0.4.0        lubridate_1.7.4   
 [16] xml2_1.2.2         knitr_1.28         config_0.3        
 [19] pkgload_1.0.2      zeallot_0.1.0      jsonlite_1.6.1    
 [22] packrat_0.5.0      rematch_1.0.1      whoami_1.3.0      
 [25] compiler_3.6.1     httr_1.4.1         backports_1.1.5   
 [28] assertthat_0.2.1   fastmap_1.0.1      lazyeval_0.2.2    
 [31] cli_2.0.2          later_1.0.0        htmltools_0.4.0   
 [34] prettyunits_1.0.2  tools_3.6.1        gtable_0.3.0      
 [37] glue_1.4.0         dplyr_0.8.3        rappdirs_0.3.1    
 [40] Rcpp_1.0.4.6       xopen_1.0.0        vctrs_0.2.0       
 [43] tigris_0.9.4       crosstalk_1.1.0.1  xfun_0.11         
 [46] stringr_1.4.0      ps_1.3.0           mime_0.7          
 [49] parsedate_1.2.0    lifecycle_0.1.0    devtools_2.3.0    
 [52] scales_1.1.0       promises_1.1.0     yaml_2.2.0        
 [55] curl_4.2           memoise_1.1.0      ggplot2_3.2.1     
 [58] rcmdcheck_1.3.3    stringi_1.4.3      maptools_0.9-8    
 [61] desc_1.2.0         e1071_1.7-3        pkgbuild_1.0.6    
 [64] attempt_0.3.0      rlang_0.4.5        pkgconfig_2.0.3   
 [67] evaluate_0.14      lattice_0.20-38    purrr_0.3.3       
 [70] sf_0.9-2           htmlwidgets_1.5.1  processx_3.4.1    
 [73] tidyselect_0.2.5   here_0.1           magrittr_1.5      
 [76] R6_2.4.1           DBI_1.1.0          pillar_1.4.2      
 [79] whisker_0.4        foreign_0.8-71     withr_2.1.2       
 [82] units_0.6-6        sp_1.3-2           tibble_2.1.3      
 [85] crayon_1.3.4       uuid_0.1-4         KernSmooth_2.23-15
 [88] rhub_1.1.1         rmarkdown_1.18     usethis_1.6.0     
 [91] grid_3.6.1         callr_3.4.3        git2r_0.26.1      
 [94] digest_0.6.25      classInt_0.4-3     xtable_1.8-4      
 [97] tidyr_1.0.0        httpuv_1.5.2       openssl_1.4.1     
[100] munsell_0.5.0      dockerfiler_0.1.3  sessioninfo_1.1.1 
[103] askpass_1.1  `

I do have my project folder initialized as a remote github repo, could this be the issue? Would appreciate your help in diagnosing what's causing this.

Thank You!

**Comment by ColinFay** [comment]
Hey, 

This bug should have been corrected via #435. 

Can you confirm by reinstalling the current version of `{golem}` from GitHub, then run: 

``` r
packageVersion("golem") == "0.2.1.9010"
unlink("tests/testthat/test-golem-recommended.R")
golem::use_recommended_tests()
devtools::test()
devtools::check()
``` 

Thanks

**Comment by jacksonschan** [comment]
@ColinFay Yes that worked, all testthat tests passed. Thank You!
